docs/en/samples/UMAPI SAMPLES.postman_collection.json

```diff
@@ -0,0 +1,1543 @@
+							"",
+							"    var header = {",
+							"        \"alg\": \"RS256\"",
+							"    };",
+							"",
+							"    var data = {",
+							"        \"exp\": Math.round(87000 + Date.now() / 1000),",
+							"        \"iss\": pm.collectionVariables.get(\"UMAPI_IMS_ORG\"),",
+							"        \"sub\": pm.collectionVariables.get(\"UMAPI_TECHNICAL_ACCOUNT_ID\"),",
+							"        \"aud\": \"https://\" + pm.collectionVariables.get(\"UMAPI_IMS\") + \"/c/\" + pm.collectionVariables.get(\"UMAPI_API_KEY\")",
+							"    };",
+							"    ",
+							"    var meta_scope = \"https://ims-na1.adobelogin.com/s/ent_user_sdk\";",
+							"    data[meta_scope] = true;",
+							"    ",
+							"",
+							"    var secret = pm.collectionVariables.get(\"UMAPI_PRIVATE_KEY\");",
+							"",
+							"    if (!secret) {",
+							"        console.log(\"Ensure the Private Key is added to both INITIAL and CURRENT VALUES in the active Postman environment's UMAPI_PRIVATE_KEY variable.\");",
+							"    }",
+							"",
+							"    console.log(data);",
+							"",
+							"    var sHeader = JSON.stringify(header);",
+							"    var sPayload = JSON.stringify(data);",
+							"    var sJWT = KJUR.jws.JWS.sign(\"RS256\", sHeader, sPayload, secret);",
+							"",
+							"    console.log(sJWT);",
+							"",
+							"    pm.collectionVariables.set(\"UMAPI_JWT_TOKEN\", sJWT);",
+							"}",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "formdata",
+					"formdata": [
+						{
+							"key": "client_id",
+							"value": "{{UMAPI_API_KEY}}",
+							"type": "text"
+						},
+						{
+							"key": "client_secret",
+							"value": "{{UMAPI_CLIENT_SECRET}}",
+							"type": "text"
+						},
+						{
+							"key": "jwt_token",
+							"value": "{{UMAPI_JWT_TOKEN}}",
+							"type": "text"
+						}
+					]
+				},
+				"url": {
+					"raw": "https://{{UMAPI_IMS}}/ims/exchange/jwt/",
+					"protocol": "https",
+					"host": [
+						"{{UMAPI_IMS}}"
+					],
+					"path": [
+						"ims",
+						"exchange",
+						"jwt",
+						""
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "GET Org users page 0",
+			"protocolProfileBehavior": {
+				"disableBodyPruning": true,
+				"disabledSystemHeaders": {
+					"content-type": true,
+					"accept": true,
+					"accept-encoding": true,
+					"connection": true
+				}
+			},
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "Accept",
+						"value": "application/json"
+					},
+					{
+						"key": "x-api-key",
+						"value": "{{UMAPI_API_KEY}}"
+					},
+					{
+						"key": "Authorization",
+						"value": "Bearer {{UMAPI_ACCESS_TOKEN}}"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "https://usermanagement.adobe.io/v2/usermanagement/users/{{UMAPI_IMS_ORG}}/{{UMAPI_PAGE}}",
```

Comment:
We should govern the API host with a variable so we can easily target stage. Otherwise there are 23 instances of the hostname in the file that one would need to manually override.